diff --git a/build/webgpu.js b/build/webgpu.js
index 1ce72679646fb617bb62b52dc7f62bf3aebc208a..8ace179aa70a52fcce34278ff33bb8201a0754a9 100644
--- a/build/webgpu.js
+++ b/build/webgpu.js
@@ -16,8 +16,8 @@ const H = "float", Ao = "float", q = "float", Fs = "float", Fo = "float", en = "
   "DensityProfileLayer"
 ), te = /* @__PURE__ */ ut(
   {
-    layer0: Re.layout.name,
-    layer1: Re.layout.name
+    layer0: (Re.layout ?? Re).name,
+    layer1: (Re.layout ?? Re).name
   },
   "DensityProfile"
 ), os = {
@@ -26,13 +26,13 @@ const H = "float", Ao = "float", q = "float", Fs = "float", Fo = "float", en = "
   sunAngularRadius: Ao,
   bottomRadius: H,
   topRadius: H,
-  rayleighDensity: te.layout.name,
+  rayleighDensity: (te.layout ?? te).name,
   rayleighScattering: At,
-  mieDensity: te.layout.name,
+  mieDensity: (te.layout ?? te).name,
   mieScattering: At,
   mieExtinction: At,
   miePhaseFunctionG: q,
-  absorptionDensity: te.layout.name,
+  absorptionDensity: (te.layout ?? te).name,
   absorptionExtinction: At,
   groundAlbedo: qt,
   minCosLight: q,
@@ -507,7 +507,7 @@ const Tt = /* @__PURE__ */ k({
     scatteringTextureCosLightSize: r
   } = j(n), u = _(s.pow2().sub(e.pow2())).toConst(), c = u.mul(
     Dt(t.w, i)
-  ).toConst(), d = _(c.pow2().add(e.pow2())), l = P(0).toVar(), h = _t().toVar();
+  ).toConst(), d = _(c.pow2().add(e.pow2())).toConst(), l = P(0).toVar(), h = _t().toVar();
   D(t.z.lessThan(0.5), () => {
     const M = d.sub(e).toConst(), N = c, b = M.add(
       N.sub(M).mul(
